cmd/fleetctl/get.go

```diff
@@ -1022,3 +1055,55 @@ func getSoftwareCommand() *cli.Command {
 		},
 	}
 }
+
+func getMDMAppleCommand() *cli.Command {
+	return &cli.Command{
+		Name:    "mdm_apple",
+		Hidden:  true, // TODO: temporary, until the MDM feature is officially released
+		Aliases: []string{"mdm-apple"},
+		Usage:   "Show Apple Push Notification Service (APNs) information",
+		Flags: []cli.Flag{
+			configFlag(),
+			contextFlag(),
+			debugFlag(),
+		},
+		Action: func(c *cli.Context) error {
+			const expirationWarning = 30 * 24 * time.Hour // 30 days
+
+			client, err := clientFromCLI(c)
+			if err != nil {
+				return err
+			}
+
+			mdm, err := client.GetAppleMDM()
+			if err != nil {
+				var nfe service.NotFoundErr
+				if errors.As(err, &nfe) {
+					log(c, "Error: No Apple Push Notification service (APNs) certificate found. Use `fleetctl generate mdm-apple` and then `fleet serve` with `mdm` configuration to turn on MDM features.\n")
+					return nil
+				}
+				return err
+			}
+
+			printKeyValueTable(c, [][]string{
+				{"Common name (CN):", mdm.CommonName},
+				{"Serial number:", mdm.SerialNumber},
```

Comment:
nit: looking at `server/service/mdm.go`, seems like `SerialNumber` can be empty, should we special case this? either not show the row or show a `-`?

Reply:
yeah, not a big deal! just a nit. I'd vote to merge without this (I know you need the parent PR merged first!)

Reply:
@noahtalerman That's correct, there will always be a serial number, I haven't found a clear excerpt in the RFC that mentions this, but the implementation in Go of the parsing of a certificate ensures it will be set: https://cs.opensource.google/go/go/+/refs/tags/go1.19.3:src/crypto/x509/parser.go;l=839-848;drc=d5de62df152baf4de6e9fe81933319b86fd95ae4